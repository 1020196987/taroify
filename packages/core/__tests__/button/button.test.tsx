import * as React from "react"
import { create } from "react-test-renderer"
import Button from "../../src/button/button"

describe("Button", () => {
  it("show", () => {
    const component = create(<Button />)
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
