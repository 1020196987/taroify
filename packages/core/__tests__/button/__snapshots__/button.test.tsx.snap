// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button show 1`] = `
<button
  className="vant-button"
/>
`;
